detection_rules/utils.py

```diff
@@ -299,6 +317,27 @@ def format_command_options(ctx):
     return formatter.getvalue()
 
 
+def make_git(*partial_args) -> Optional[Callable]:
```

Comment:
```suggestion
def make_git(*persistent_args) -> Optional[Callable]:
```

Reply:
seemed similar naming to `functools.partial`
i don't have strong feelings here though.

`prefix_args`?

Reply:
I like prefix_args

Reply:
816619574367cdde9addc2f5fc4a3a60c7d9a1ca